pkg/component/logging/vali/templates/vali-config.yaml.tpl

```diff
@@ -1,4 +1,3 @@
-auth_enabled: {{ .AuthEnabled }}
```

Comment:
You also have to drop https://github.com/gardener/gardener/blob/d4a4cbada3cd9b18a3e9ea83d566059b741933ab/pkg/component/logging/vali/vali.go#L145

Reply:
done in previous commit